lib/commands/toggleList.test.js

```diff
@@ -343,7 +343,7 @@ const valueNestedListSelection = {
     ],
 };
 
-const valueNestedListSelectionParentNotList = {
+const valueNestedListSelectionParentEditor = {
```

Comment:
I see both the name and content of the test is changed. 
Is the old test no longer valid (not desired behaviour). I'm kind of scared to change some intention of behaviour we recreated before.